### PR TITLE
GraphQL coverage for shopper assistance

### PR DESCRIPTION
**Pull request by sathiyaaa**
## Purpose of this pull request

This pull request (PR)  covers #8281 

## Affected DevDocs pages

-  /graphql/mutations/generate-customer-token-as-admin.html

## Links to Magento source code

-  magento/architecture#414

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Great addition!

File: src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+```
+
+## Input attributes
+
+The `generateCustomerTokenAsAdmin` mutation requires the following input attribute:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`customer_email` | String | The customer's email address
```
Comment (rogyar) [inline src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md:51]: Please note, `customer_email` is a required attribute

```suggestion
`customer_email` | String! | The customer's email address
```

File: src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Output attributes
+
+The `generateCustomerTokenAsAdmin` mutation returns a valid customer token as the output.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`customer_token` | String | The customer token
```
Comment (rogyar) [inline src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md:59]: Please note, `customer_token` is a required attribute 

```suggestion
`customer_token` | String! | The customer token
```

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @sathiyaaa. Good job, please check my minor recommendations below. 

@BarnyShergold sorry for jumping in. To some extend, I took part in the schema grooming and feel responsible for the GraphQL coverage :) 

Thanks

File: src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md (hunk 1 above)
Comment (sathiyaaa) [reply src/guides/v2.4/graphql/mutations/generate-customer-token-as-admin.md:51]: Ah yes! My bad. Thanks. @rogyar

**Review by rogyar — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Review by keharper — APPROVED** [review]

**Pull request merged**